doc/onboarding.md

```diff
@@ -32,33 +33,38 @@ onboarding session.
 ## Local setup
 
   * git:
-    * make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
-    * usually PR from your own github fork
+    * Make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
+    * Always continue to PR from your own github fork
```

Comment:
I don't think this is universally true - sometimes there is value in having a PR off a branch that others can easily edit.